internal/xds/translator/listener.go

```diff
@@ -775,3 +780,35 @@ func buildHeadersWithUnderscoresAction(in *ir.HeaderSettings) corev3.HttpProtoco
 	}
 	return corev3.HttpProtocolOptions_REJECT_REQUEST
 }
+
+func buildForwardClientCertDetailsAction(in *ir.HeaderSettings) hcmv3.HttpConnectionManager_ForwardClientCertDetails {
+	if in != nil {
+		switch in.ForwardClientCertDetails {
+		case ir.ForwardClientCertDetailsSanitize:
+			return hcmv3.HttpConnectionManager_SANITIZE
+		case ir.ForwardClientCertDetailsForwardOnly:
+			return hcmv3.HttpConnectionManager_FORWARD_ONLY
+		case ir.ForwardClientCertDetailsAppendForward:
+			return hcmv3.HttpConnectionManager_APPEND_FORWARD
+		case ir.ForwardClientCertDetailsSanitizeSet:
+			return hcmv3.HttpConnectionManager_SANITIZE_SET
+		case ir.ForwardClientCertDetailsAlwaysForwardOnly:
+			return hcmv3.HttpConnectionManager_ALWAYS_FORWARD_ONLY
+		}
+	}
+	return hcmv3.HttpConnectionManager_SANITIZE
```

Comment:
Should we keep the envoy default of `SANITIZE` or use something like `SANITIZE_SET` if client-cert auth is enabled?

Reply:
resolving this since we will not enabling this by default